TeamCode/build.gradle

```diff
@@ -19,7 +19,9 @@ repositories {
 }
 
 dependencies {
-implementation 'org.openftc:easyopencv:1.4.1'
- 
+    implementation 'org.openftc:easyopencv:1.4.1'
+    compile group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
```

Comment:
Is it not possible to import only the linear library?

Reply:
unfortunately not

Reply:
Pretty sure 'compile' was replaced with 'implementation'